core/iwasm/compilation/aot_emit_aot_file.c

```diff
@@ -2098,6 +2097,12 @@ aot_emit_custom_sections(uint8 *buf, uint8 *buf_end, uint32 *p_offset,
         const uint8 *content = NULL;
         uint32 length = 0;
 
+        if (strcmp(section_name, "name") == 0) {
+            aot_emit_name_section(buf, buf_end, p_offset, comp_data, comp_ctx);
```

Comment:
Should also add `*p_offset = offset` before calling aot_emit_name_section? Suppose emit other section first, then emit name section, `*p_offset` isn't updated?